services/can-watchdog/can-watchdog.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+import glob # For expanding wildcard filepaths
+import jeepney # For interactions with dbus
+from subprocess import call
+
+print('CAN Watchtog has started')
+print('Getting all can rx counters')
+rx_paths = glob.glob('sys/class/net/can*/statistics/rx_bytes')
```

Comment:
I'm not sure how to fix, but this will break if we were to name the can interfaces something other then can*.

Could maybe query netlink directly and find all can interfaces to check. libnl looks useful. Not sure though.